TabnineSelfHosted/src/main/java/com/tabnineSelfHosted/statusBar/TabnineSelfHostedStatusBarWidget.kt

```diff
@@ -47,6 +49,12 @@ class TabnineSelfHostedStatusBarWidget(project: Project) :
                     update()
                 }
             )
+
+        CompletionsStateNotifier.subscribe(object : CompletionsStateNotifier {
```

Comment:
is this enabled only in self hosted?
just making sure you haven't forgot anything by mistake

Reply:
I actually handled SAAS, look at my comment